app/Jobs/CirrusSearch/ElasticSearchIndexInit.php

```diff
@@ -6,6 +6,13 @@
 
 class ElasticSearchIndexInit extends CirrusSearchJob
 {
+    private $cluster;
+
+    public function __construct( int $wikiId, string $cluster = 'all' ) {
```

Comment:
Question: At which point will someone pass a different value than `all` here? Asking as I wonder why we don't just hardcode it in `getQueryParams`?